plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergConfig.java

```diff
@@ -91,4 +92,22 @@ public IcebergConfig setMaxPartitionsPerWriter(int maxPartitionsPerWriter)
         this.maxPartitionsPerWriter = maxPartitionsPerWriter;
         return this;
     }
+
+    /**
+     * In case of some queries / tables, retrieving table statistics from Iceberg
```

Comment:
This probably belongs in docs

Reply:
It's fine to have this as a code comment as well, but that's not a javadoc. A javadoc should describe what a thing is, but that's not needed here, as the config is self-explanatory.

Please convert it into `/*` or `//` ordinary comment.

Reply:
Sure, can update to a usual comment. I'll also add a note on this property in the iceberg docs.

Reply:
Let's make docs a followup, as there will surely be some back and forth on the wording.
Actually, i wouldn't document that, as this is an escape hatch only, and we should rather fix the underlying problem(s) instead.
For example, `hive.table-statistics-enabled` remains undocumented.